### ISSUE TITLE
seqkit shuffle --two-pass adds extra line return

### ISSUE THREAD

**Issue by nick-youngblut** [shenwei356/seqkit#173]
```
$ head seqs.fasta
>f9b7e3d269fe4babaec34f5e196a0d27
ATGACATACCCAATAGCAACAATAATATTCCTAGCACTAATAGCATTCGTAATACTACTAGGATTCAGCGCAGCAAGATGGAAAGCAGGAGACATGAGCCACATAGAAGAATGGGGACTAGGAGGAAGAAGATTCGGAACACTAATAAGCTGGTTCCTAATAGGAGGAGACATATACACAGCATACACATTCATAGCAATACCAGCACTAATGTTCGGAGCAGGAGCAATAGGATTCTTCGCAGTACCATACACAGTAGTATGCTACCCAATACTATACATAGTATTCCCAAAACTATGGAAAGTAGCACACAAATACGGATACATAACAGGACCAGAATTCGTAAGAGGAAGATACGGAAACAAATACCTAGGACTAGCAATAGGACTAACAGGAATATTCGCAACAATGCCATACATAGCACTACAACTAATAGGAATGCAAACAGTAATAGGAGCACTAGGAATACCAGGACACCTACCACTAATAATAGCATTCGGAGTACTAGCAGCATTCACATACACAAGCGGACTAAGAGCACCAGCAATGATAGCAATAGTAAAAGACCTACTAATATACATAACAGTAATAGCAGCAGTAATAGTAATACCAGCAGAACTAGGAGGATTCAGCGCAATATTCGCAAAAATAAGCCCAGCAAAACTACTACTAGCAGCACCACCAGCAGGAAGCACAGGAGCATACACAGGATACGCAACACTAGCACTAGGAAGCGCAATGGCACTATTCCTATACCCACACAGCACAACAGCAATACTAAGCGCAAGCGGACCAAGA
>7aad99eff09746dd9c5dc763c11fc43d
GTAAGCGACTTCCTACTACAACTACAAAACCTAGTAAGAACATACAAAAGCGAAAGCGAAGCACTAACAGTACTAGACGGAGCAAACCTAGACCTAAGAAGAGGAGAAATAGTAGCACTAGTAGCACCAAGCGGAAGCGGAAAAAGCACACTACTACACCTAGCAGGACTACTAGAAAAACCAGACGGAGGAGCAGTAATAATAAACGGAGCAGACGCAGGAAAACTAGGAGACACAGAAAGAACAAACATAAGACTACACACAATAGGAATAGTATACCAATTCCACCACCTACTAGCAGAATTCAGCGCACTAGAAAACGTAGTACTACCACAAATGATAGCAGGAAAAAGCAAAGCAGAAGCAGAAAAAAGAGCAATGGAACTACTAACAAAACTAGGACTAGCAGCAAGAGCAGACCACCTACCAGGAAAACTAAGCGGAGGAGAACAACAAAGAGTAGCAATAGCAAGAGCACTAGCAAACGCACCAGCACTACTACTAGCAGACGAACCAACAGGAAACCTAGACGTAGCAACAGCAGAAATAGTATTCAGCGAACTACTAAGAATAGTAAGAAGCGAAAACGTAGCAGCACTAATAGCAACACACAACCCAGAACTAGCAAAAAGAATGGACAGACAACTAACACTAGTAGGAGGAAAACTAATANNN
>fff1e7d7226f475ebfed96329efa008e
ATGCCAGCACCAAGCCACAGATTCACAGAAGCACCAGACATATGGCCACCACCAGGAACAGGAGTACCAGAAACAGACATGCCAGGATGGGGAACAACAATAGGAAGAGGAATAAGAGGAACATGCCCAAGATGCGCAAAAGCACCAATATTCAACGGATACCTAAAAGTACACACAGTATGCGCAAGCTGCGGAGCACCACTAGGAGACATGCCAAGCGACGACGCACCACCATACATAGCAATGCTAGTAGTACTACACGTACTAGCATTCTTCGTAGTAATGATATTCAGATACGGATGGCAACCAAGCCCATACGAATACGGAGCAATGCTACTAGTACTAGTAGCAGCATGCATGGTAGGACTAAGAATGGCAAAAGGAGCAACAATAGGAATACTACTAAAACTAGGACTAAAAAGAGCAGTACTAAACNNN
>89571a38a8d34548b44c54987286c97d
ATGAGCATAGCACTACTAGCAGTACTATTCCCACTACTAACAGCAATACCACTAGCACTAGTAGGAAGCGCAACAAGAGCAAGCTACGTAAACCTAGCACTAAACACAGTAACATTCGCACTAACACTAGGAGTAGGATTCAGCCCAGGAAACGGAATGCTACACGCAGACAACCTAGGAATAATATTCGGAATACTAAGCGGATTCGTAGGATTCACAACAAGCATAAACAACATAGGATTCATAAAAAGCGAACAAGGAAGAAAAAGCATAAGAAGATGGAGACTATACCACGCAATGTGCCAAGTAATGCTAGGAACAACACTACTAGGACTATACGCAGACAACATAGGACTACTATGGGTAGCAGTAGAAGGAGCAACAATAAGCGCAACACTAGGAGTAAGCCTAAGCAGAACAGCAAGCGCACTAGAAGCAGCATGGAAATACTTCATACTAGGAGGAGTAGGAATAGCACTAGCACTATTCGGAATAATGCTAACATACCTAGCAGCACAACCAGCACTAGGACCAGGAATAGCAGCAATGAGCTTCACAGCACTAACACAACACGCAGCACAACTAGACGGAAGCCTAATGACACTAGCATTCGTATTCCTACTATTCGGATTCGGAACAAAAGCAGCACTAGTACCACTACACGGATGGCTACCAGACGCATACGCAGAAGGACCAATACCACTAACAGCAACACTAAGCGGACTAATGCTAAACGTAGCACTACTAGCACTACTAAGATTCAGACACCTAATGAGCGCAAACGCAGCAGCAGGAGCAGGAGCAATGCACCCAGGACCATTCCTACTAATACTAGGACTAGCAAGCCTATTCCTAACAGGATTCAGCCTATGGAGAAGAAGAGACAGCAGAAGATTCTTCGGATTCAGCAGCATAGAACACAGCGGAATAGCAATATTCGCATTCGGAATAGGAGGACCAGCAATAATATTCGGAGGACTACTACACATGCTACTACACACACTACTAAAAAGCGCACTATTCCAAGCACTAACAAAAGCAGCACTAGCAAGAGGAAGCGCAGCACCAGGAAACAGCGGATTCAGCCACCTAAGAGGAATGATAGCAACAAGCAAACAAACAGGATGGCTACTAGTAATATGCATATTCGCACTAACAGGACTACCACCAAGCGGACTATTCACAAGCGAATACCTAATAATAAGCGAAACAATAATAAGAGACCCATACTACAGCCTACCACTAGGAGCAGGACTACTACTATGCGCAATAGCAATAATAAGACAAGTAGGACCAGTAGGATTCGCACCAGCACCAGCAAGCGCAAAACCACAAAAAGCAGGAGCAGACGTAATAACAATATACGCACACCTAATACTAGTAGTACTACTAGCATTCGCAATGCCAGTAACACTAGTAACACTACTAAGCCACATAGCAGAAGCACTACAANNN
>44a8e1f5c2ea4c788ed6a72e3c3dc965
ATGAAACAAGTAAAAAACGACCCATTCGTAACATTCGGAGACTGGCTAACAGAAGCAGAAAGCAGCGAACCAAACGACCCAAACGCAATGATACTAGCAACAGCAAGCGCAGAAGGAAAACCAGCAGCAAGAACAGTACTACTAAAAGCATGGGACGACAAAGGATTCGTATTCTACACAAACCTAGAAAGCAGAAAAAGCCAAGAACTAAGACAAAACCCACACGCAGCACTACTATTCTACTGGAAAACACTACACAGACAAGTAAGAATAGAAGGAACAATAAGCCTAGTAAGCGACGCACAAGCAGACGAATACTACGCAAGCAGACCAAGAGGAAGCAGAATAGGAGCATGGGCAAGCGCACAAAGCAAACCACTACCAGACAGAGAAACATTCGAAGCAAAATTCGAAGAATTCGAAGCAAAATTCCCAGGAGACACAATACCAAGACCACCATTCTGGAGCGGATGGAGAGTAACACCAGACTACCTAGAATTCTGGCAAGACGTAAAATACAGACTACACGAAAGAACAACATACACAAGAGACGGAAACGAATGGATAACAGGAAAACTATACCCANNN
```

```
$ seqkit shuffle --two-pass seqs.fasta | head
[INFO] create and read FASTA index ...
[INFO] read sequence IDs from FASTA index ...
[INFO] 39271 sequences loaded
[INFO] shuffle ...
[INFO] output ...
>355a3df576af4fddb779d9bd36569322
ATGAGCGCACAAAGCTGCCTAGTATGCCAACAAGGAAGCGCAATACTAGACTGGAAAAAACCAGAAAGCCTAAAACCATTCCTAAACGCAAGAGGAGGAGTAAAAAACAAAAACCAAAGCGGACTATGCAACACACACCACGCATTCATGAAAAAACAAGTAGACAGAGCAAGAAAAATAGCACTACTACCATACCCAACAAGCNNN

>a90b68e5adf742ef82713f3230b8fc78
ATGGGACTAATGACAGAAAAACACGGAAAATTCTTCATAGTAATGGGAATACTAATACTAGTACTAAGCATATACATAGTAAGCAAAGACCCAATAATAGGAGGAGTAGGAGTACTAATAGGACTATGGAACATAGTAATGGGATACAGAACAAGCAAAGGAAAAACATTCTTCCCAAGCGACGACGACNNN

>2cf35c97ea044be8b25b9de97a8b5a3c
ATGAGCGCACCAGGAGCAGAACTAGCAAGAGAACTACTAAGAGTAAGAGACCTAATAGACAGAGACTACGCAGAAGAACTAGACATAGCAGCACTAGCAAGAAGCGCATGCGTAAGCCAAGCACACTTCAGAAGAACATTCAAACAAGCATTCGGAGAAACACCATACAGATACCTAATGACAAGAAGACTAGAAAGAGCAAAAGCACTACTAAGAGCAGGAGAACTAAGCGTAACAGAAGTATGCCTAGCAGTAGGATTCACAAGCCTAGGAAGCTTCAGCACACAATTCAGAAGATTCGTAGGAGAAAGCCCAACAGCATACAGAAGAAGAGCAGCAACACTACCACCAGTACCAATACCAGCATGCTACCTAAGAATATGGACAAGACCAAGAGAANNN

```

No extra line returns are added if I exclude the `--two-pass` param.

I'm using `seqkit   0.14.0  bioconda`

**Comment by shenwei356** [comment]
Thanks, it's fixed.

- [seqkit_darwin_amd64.tar.gz](https://github.com/shenwei356/seqkit/files/5649804/seqkit_darwin_amd64.tar.gz)
- [seqkit_linux_amd64.tar.gz](https://github.com/shenwei356/seqkit/files/5649805/seqkit_linux_amd64.tar.gz)

**Comment by shenwei356** [comment]
Hmm, the fix introduced a new bug #209.

Actually, we can just use `seqkit seq` to remove extra `\n`.
